hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -798,6 +798,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=20_000_000", "+en_jitter=1"]
     }
+    {
+      name: chip_sw_keymgr_sideload_kmac
+      uvm_test_seq: chip_sw_keymgr_sideload_kmac_vseq
```

Comment:
we don't have `chip_sw_keymgr_sideload_kmac_vseq` yet, use the default SV sequence for now.
```suggestion
      uvm_test_seq: chip_sw_base_vseq
```